html/usage.html

```diff
@@ -70,20 +85,49 @@
                 }
             });
 
-            var datasets = [{
-                data: libraries.map((lib) => {
-                    var filteredUsageData = usage.filter((d) => d.library === lib);
-                    return _.reduce(filteredUsageData, (memo, d) => {
+            // Filter the library usage to get its total_usage as well as color and category
+            let filteredUsageData = libraries.map((lib) => {
+                    var usageData = usage.filter((d) => (d.library === lib));
+                    // Set the color for the library
+                    const color = cpplibraries.includes(lib) ? 'rgba(255, 0, 0, 0.2)'
+                                                    : rustlibraries.includes(lib) ? 'rgba(0, 0, 255, 0.2)' 
+                                                    : 'rgba(127, 127, 127, 0.2)';
+                    // Set the category for which the library belongs
+                    const category = cpplibraries.includes(lib) ? 'cpp'
+                                                    : rustlibraries.includes(lib) ? 'rust' 
+                                                    : 'others';
+                    return {
+                        'library': lib,
+                        color,
+                        category,
+                        'usage': _.reduce(usageData, (memo, d) => {
                             return memo + parseInt(d.times_used);
-                        }, 0);
-                    }),
-                backgroundColor: colors
+                            }, 0),
+                        }
+                    })
+            
+            // Get the libraries to display on chart 
+            // and that which will not be on the chart
+            let dataOnChart = filteredUsageData.filter(x => x.usage >= minRange);
+            let dataNotOnChart = filteredUsageData.filter(x => x.usage < minRange)
+            console.log(dataOnChart)
+            
+            // Display libraries not on chart as un-ordered list
+            dataNotOnChart.forEach( info => {
+                $('#notOnChart').append(`<li> ${info.library} (${info.category}) - ${info.usage} </li>`)
```

Comment:
I didn't know this was possible, but it's been implemented for a while in browsers I see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals - thanks for doing this!